### PR TITLE
Fix dangling batch normalization for twisted Edwards curves

### PR DESCRIPTION
**Pull request by weikengchen**
## Description

It is discovered that Pedersen commitments are failing. After some debugging, we confirm that it is because `normalize_batch`, different from `batch_normalize`, returns the results instead of applying the mutation directly.

This PR fixes so.

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [x] Targeted PR against correct branch (master)
- [x] Linked to Github issue with discussion and accepted design OR have an explanation in the PR that describes this work.
- [x] Re-reviewed `Files changed` in the Github PR explorer

N/A: 
- [ ] Wrote unit tests
- [ ] Updated relevant documentation in the code
- [ ] Added a relevant changelog entry to the `Pending` section in `CHANGELOG.md`

### CODE REVIEW

**Comment by weikengchen** [comment]
There appears to be another bug regarding Montgomery representations. Will do in a separate PR.
